.gitignore

```diff
@@ -2,8 +2,10 @@
 /build
 /certs
 /coverage
+/dist
 /node_modules
 /publish
 .DS_Store
 /storybook-static
 .vscode
+*.local
```

Comment:
The files in the `locales/` folder appear as untracked. Can we add that to .gitignore?
![image](https://github.com/ansible/ansible-ui/assets/43621546/1afe041a-5580-408d-8d3b-3f69071dbd04)

Reply:
the locales had to move into the public folder for vite to automatically pick them up. you should not need a locales folder at the root.